app/scripts/metamask-controller.js

```diff
@@ -254,6 +266,22 @@ module.exports = class MetamaskController {
     this.configManager.useEtherscanProvider()
     extension.runtime.reload()
   }
+
+  buyEth (address, amount) {
+    if (!amount) amount = '5'
+
+    var network = this.idStore._currentState.network
+    var url = `https://buy.coinbase.com/?code=9ec56d01-7e81-5017-930c-513daa27bb6a&amount=${amount}&address=${address}&crypto_currency=ETH`
+
+    if (network === '2') {
+      url = 'https://testfaucet.metamask.io/'
+    }
+
+    chrome.tabs.create({
```

Comment:
replace with `extension.tabs.create`.

Also:  Why are we even bothering to do this in the background process? Why not just create this tab from the `actions.js` file?